sonic_y_cable/credo/y_cable_credo.py

```diff
@@ -1469,8 +1475,12 @@ def get_firmware_version(self, target):
                     status = self.send_vsc(vsc_req_form)
 
                     if status != YCable.MCU_EC_NO_ERROR:
-                        self.log_error('Get firmware version error (error code:0x%04X)' % (status))
-                        return None
+                        ''' should at least return local side fw version if nic is offline'''
+                        if status == 0x1A and (read_side == target):
```

Comment:
@xinyulin 0x1A can you name this status value

Reply:
Sure, I made the correction in the last commit. Please review it.